dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -23,7 +23,7 @@ before(function () {
 
 describe("As a user, I expect the search functionality on the /companies page to adjust to the selected dropdown filters", () => {
   it(
-    "The framework filter should contain LKSG and SFDR  even though they are not yet implemented, and synchronise " +
+    "The framework filter should contain LKSG and SFDR even though they are not yet implemented, and synchronise " +
```

Comment:
Why not SME? Also syncronize**d**, I'd say

Reply:
I'd like to ask Varenya on thursday if we shall include sme in this dropdown list. If yes, then it will be needed here in the test.